jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -110,7 +110,8 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_EXEC, "exec")
 #if !defined (CONFIG_DISABLE_ES2015_TYPEDARRAY_BUILTIN)
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_FROM, "from")
 #endif
-#if !defined (CONFIG_DISABLE_ARRAY_BUILTIN)
+#if !defined (CONFIG_DISABLE_ARRAY_BUILTIN) \
+|| !defined (CONFIG_DISABLE_ES2015_TYPEDARRAY_BUILTIN)
```

Comment:
wrong indentation

Reply:
That code was generated by `gen-magic-strings.py`.